### PR TITLE
Component/vote form

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
- create vote form
- add form to proposal view

### CODE REVIEW

File: src/app/components/VoteForm/VoteForm.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    neutral: '=',
+    remarks: '?',
+    concerns: '!',
+  }
+  const voteTypeColors = {
+    pro: '#65ac95',
+    neutral: 'var(--c-gray-100)',
+    remarks: '#b6a55c',
+    concerns: '#b95730',
+    none: 'var(--c-gray-200)',
```
Comment (derjubu) [inline src/app/components/VoteForm/VoteForm.tsx:61]: Why not variables for all colors?

File: src/app/pages/ProposalView/ProposalView.tsx [hunk 2]
```diff
@@ -41,6 +46,11 @@ export default function ProposalView(): JSX.Element {
     text: `${category} (${votes[category as keyof typeof votes].length})`,
   }))
 
+  function handleVoteSubmit(newVote: Vote) {
+    console.log(newVote)
```
Comment (derjubu) [inline src/app/pages/ProposalView/ProposalView.tsx:50]: console.log 🕵️

**Review by derjubu — APPROVED** [review]
LGTM 💯 
Just some small comments

File: src/app/components/VoteForm/VoteForm.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  }
+
+  function handleCancel() {
+    setText('')
+    setVoteType('none')
+    setSubmitError(false)
+    onCancel()
+  }
+
+  const textDiff = MAX_DESCRIPTION_LENGTH - text.length
```
Comment (janhendrikstruve) [inline src/app/components/VoteForm/VoteForm.tsx:48]: Maybe consider a naming like "descriptionLeft", "descriptionRamaining" or "textRemaining"? :)

**Review by janhendrikstruve — COMMENTED** [review]
Great Code, just a minor naming thing :)

**Pull request merged**